debian/changelog

```diff
@@ -1,4 +1,63 @@
-ubuntu-advantage-tools (27.1) impish; urgency=medium
+ubuntu-advantage-tools (27.2) impish; urgency=medium
+
+  * d/control:
```

Comment:
Missing something to capture the postinst changes:
```suggestion
  * d/tools.postinst: run status.json schema patch script to avoid non-root status errors
  * d/control:
```